### ISSUE TITLE
Gazer 2.3.0 is returning SSL errors with --no-verify-ssl  option

### ISSUE THREAD

**Issue by jsill14** [looker-open-source/gzr#56]
gzr returns below error message in 0.2.3 when --no-verify-ssl is specified. Removing gazer-0.2.30 and switching back to gazer-0.2.28 resolved the issue.  

**ERROR: SSL Certificate could not be verified
Do you need the --no-verify-ssl option or the --no-ssl option?**

**Comment by drstrangelooker** [comment]
I'm looking at this now. Just to verify, you are attempting to connect to an instance with a self-signed certificate?

**Comment by drstrangelooker** [comment]
I used to connect to the API and log in once. I did this to test the SSL connection and give a reasonable error message, then I would query what versions of the API ver available. Then I would disconnect and reconnect with the proper version of the API. This required two connections, and the Looker server would undergo some overhead to create a new session for the API, only for it to be disconnected without doing anything.

The ruby sdk provided by Looker for the API is build using a tool called Sawyer to make rest calls to the server. Sawyer is using a tool called Faraday to actually handle the http transport.

Instead of instantiating the API, I first create a Sawyer connection and check the resource at `/versions` which gives a json document with the supported versions of the API as well as the current "best" version. I do these calls by configuring Faraday and Sawyer directly.

I made a mistake in this case. I configured a new instance of Faraday and passing in the SSL config. Then I configured an instance of Sawyer. But instead of proving Sawyer the Faraday instance I already created, I accidentally let Sawyer create a new Faraday instance with default settings.

This has been fixed now and I use the Faraday instance with the proper SSL config.
